doc/backend_vars.asciidoc

```diff
@@ -19,31 +11,23 @@ IPMI_PASSWORD;;;
 IPMI_USER;;;
 |====================
 
-.KVM2USB backend
-[grid="rows",format="csv"]
-[options="header",cols="^m,^m,^m,v",separator=";"]
-|====================
-Variable;Values allowed;Default value;Explanation
-HWSLOT;;;
-ISO;;;
-|====================
-
 .QEMU backend
 [grid="rows",format="csv"]
 [options="header",cols="^m,^m,^m,v",separator=";"]
 |====================
 Variable;Values allowed;Default value;Explanation
-ADDONS;boolean;0;Enables use of ISO_$i variables
+$k;;;
+$var;;;
```

Comment:
Yeah, this is not really useful. I'm taking notes, will update the script later. ( note to #339 )